hack/asciidoc-flows-gen.sh

```diff
@@ -1,38 +1,44 @@
 #!/bin/sh
 
 ADOC=docs/flows-format.adoc
-MD_SOURCE=https://raw.githubusercontent.com/netobserv/network-observability-console-plugin/main/web/docs
+SOURCE=controllers/consoleplugin/config/static-frontend-config.yaml
 
 # Header
 echo "// Automatically generated by '$0'. Do not edit, or make the NETOBSERV team aware of the editions." > $ADOC
 cat ./hack/flows-format-header.adoc >> $ADOC
 echo -e "\n" >> $ADOC
 
-# Labels
-kramdoc -o - <(curl -fsSL ${MD_SOURCE}/interfaces/Labels.md) \
-  | sed -r 's/^= /== /' \
-  | sed -r 's/Interface: //' \
-  | sed -r '/Properties/d' \
-  | sed -r 's~xref:.*FlowDirection\.adoc\[`FlowDirection`\]~`FlowDirection` (see the following section, Enumeration: FlowDirection)~' >> $ADOC
+echo -e '[cols="1,1,3,1,1",options="header"]' >> $ADOC
+echo -e '|===' >> $ADOC
+echo -e '| Name | Type | Description | Filter ID | Loki label' >> $ADOC
 
-# Fields
-echo -e "\n" >> $ADOC
-kramdoc -o - <(curl -fsSL ${MD_SOURCE}/interfaces/Fields.md) \
-  | sed -r 's/^= /== /' \
-  | sed -r 's/Interface: //' \
-  | sed -r '/Properties/d' \
-  | sed -r 's~xref:.*InterfaceDirection\.adoc\[`InterfaceDirection`\]~`InterfaceDirection` (see the following section, Enumeration: InterfaceDirection)~' >> $ADOC
-
-# FlowDirection enum
-echo -e "\n" >> $ADOC
-kramdoc -o - <(curl -fsSL ${MD_SOURCE}/enums/FlowDirection.md) \
-  | sed -r 's/^= /== /' \
-  | sed -r '/Enumeration Members/d' >> $ADOC
+nbfields=$(yq '.fields | length' $SOURCE)
 
-# InterfaceDirection enum
-echo -e "\n" >> $ADOC
-kramdoc -o - <(curl -fsSL ${MD_SOURCE}/enums/InterfaceDirection.md) \
-  | sed -r 's/^= /== /' \
-  | sed -r '/Enumeration Members/d' >> $ADOC
+for i in $(seq 0 $(( $nbfields-1 )) ); do
+  entry=$(yq ".fields | sort_by(.name) | .[$i]" $SOURCE)
+  name=$(printf "$entry" | yq ".name")
+  type=$(printf "$entry" | yq ".type")
+  desc=$(printf "$entry" | yq ".description")
+  filter=$(printf "$entry" | yq ".filter")
+  if [[ "$filter" == "null" ]]; then
+    filter=$(yq ".columns[] | select(.field == \"$name\").filter" $SOURCE | sed 's/null//')
+    if [[ "$filter" == "" ]]; then
+      filter="n/a"
+    else
+      filter="\`$filter\`"
+    fi
+  fi
```

Comment:
```suggestion
  else 
    filter="\`$filter\`"
  fi
```

![image](https://github.com/netobserv/network-observability-operator/assets/91894519/62174d6f-a244-4629-8bea-2a01289de394)